src/utils/RouteUtils.js

```diff
@@ -91,7 +100,32 @@ async function getRoute(destinationName, end, start, departureTimeQuery, arriveB
     return busRoute;
 }
 
+async function getDetailedRoute(destinationName, end, start, departureTimeQuery, arriveBy) {
```

Comment:
Please rename all "detailedRoute" to "routeV2" (for example: getDetailedRoute -> getRouteV2) or something similar so we know the object this function returns adheres to the V2 specification.